universum/modules/artifact_collector.py

```diff
@@ -153,7 +155,7 @@ def preprocess_artifact_list(self, artifact_list, ignore_already_existing=False)
 
     @make_block("Preprocessing artifact lists")
     def set_and_clean_artifacts(self, project_configs, ignore_existing_artifacts=False):
-        artifact_list = []
+        artifact_list: List[Dict] = []
```

Comment:
In order to make this annotation work you need to annotate the method

Reply:
this looks a bit random. I wonder how it got there. I'll remove it for now, since it's not a dependency for other changes